### DISCUSSION TITLE
[Request] Combined g-force graph page for rides

### DISCUSSION THREAD

**Discussion by keithstellyes** [OpenRCT2/OpenRCT2#15805] [Feature requests]
If you're a bit new to the game, designing roller coasters can be a bit of a challenge trying to balance both lateral and vertical G's, and the UI can be not too great when trying to follow a test ride and flipping through both tabs. So I think it would be nice to have some sort of combined view. Whether it be both graphs side-by-side, or two lines drawn on the same graph.

I'm thinking like an additional tab, like "Comb. G's", how it looks now:

![Screenshot from 2020-10-31 09-21-58](https://user-images.githubusercontent.com/13317074/97784343-1bf5ce80-1b5b-11eb-8754-a6cb5cf82eb0.png)

**Comment by ocalhoun6** [comment]
Or maybe a 'show all' tab at the bottom, that will make the window big enough to hold all four graphs and display all four at the same time? (And then the window returns to normal size if you select one of the single graph buttons or change to a different tab of the window.)
